scripts/delayed.js

```diff
@@ -121,3 +121,36 @@ if (document.querySelector('body.resource-center')
         || document.querySelector('meta[name="media-type"]')?.getAttribute('content') === 'Info Sheets')) {
   await loadYMAL();
 }
+
+/**
+ * This method triggers the modal if user scrolls to config[value]  percentage of the visible page
+ * @param {Object} config
+ */
+async function openOnScroll(config) {
+  const value = parseInt(config.value, 10);
+  const target = document.createElement('div');
+  const documentHeight = document.documentElement.scrollHeight;
+  const triggerPosition = (value / 100) * documentHeight;
+  target.style.position = 'absolute';
+  target.style.top = `${triggerPosition}px`;
+  target.classList.add('test');
+  document.body.appendChild(target);
```

Comment:
Is it still required in case of delay? cant we leverage document.documentElement

Reply:
You should be leverage threshold and document.documentElement instead of adding your new div